cmd/xl-v1-multipart.go

```diff
@@ -40,7 +41,7 @@ func (xl xlObjects) getMultipartSHADir(bucket, object string) string {
 }
 
 // isUploadIDExists - verify if a given uploadID exists and is valid.
-func (xl xlObjects) isUploadIDExists(bucket, object, uploadID string) bool {
+func (xl xlObjects) isUploadIDExists(ctx context.Context, bucket, object, uploadID string) bool {
```

Comment:
There is no need to pass `ctx` to `isUploadIDExists`. It is not used in any way.

Reply:
context will be added to `isObject` call within `isUploadIDExists` in the next PR.